### PR TITLE
style for desktop

### PR DESCRIPTION
**Pull request by LazarevSlava**
# Style for desktop

- styles

## Checklist

- [x] Linter checks
- [x] Code style checks

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -17,14 +17,22 @@ function Footer() {
         </Button>
       </div>
       <div className={style.down_block}>
-        <Logo fill="white" />
-        <LinksBlock
-          title="Features"
-          linkList={['Link', 'Shortening', 'Branded Links', 'Analytics']}
-        />
+        <div className={style.logo}>
```
Comment (michey85) [inline src/components/Footer.jsx:20]: Why not just add className to `<Logo />`?

File: src/components/Header.jsx [hunk 2]
```diff
@@ -19,7 +19,18 @@ function Header() {
         <button className={style.btn}>Resources</button>
         <hr className={style.hr} />
         <button className={style.btn}>Login</button>
-        <button className={`${style.btn} ${style.btn_sign}`}>sign Up</button>
+        <button className={`${style.btn} ${style.btn_sign}`}>Sign Up</button>
```
Comment (michey85) [inline src/components/Header.jsx:22]: I think you installed a library to handle more than 1 classNames. Use it here.

File: src/components/Header.jsx [hunk 3]
```diff
@@ -19,7 +19,18 @@ function Header() {
...
+      </div>
+      <div className={style.menu_big_screen}>
+        <div>
+          <p>Features</p>
+          <p>Pricing</p>
+          <p>Resources</p>
+        </div>
+        <div>
+          <p>Login</p>
+          <button className={`${style.btn} ${style.btn_sign}`}>Sign Up</button>
```
Comment (michey85) [inline src/components/Header.jsx:32]: And here.

File: src/components/Header.jsx [hunk 4]
```diff
@@ -19,7 +19,18 @@ function Header() {
...
         <hr className={style.hr} />
         <button className={style.btn}>Login</button>
-        <button className={`${style.btn} ${style.btn_sign}`}>sign Up</button>
+        <button className={`${style.btn} ${style.btn_sign}`}>Sign Up</button>
+      </div>
+      <div className={style.menu_big_screen}>
+        <div>
+          <p>Features</p>
+          <p>Pricing</p>
+          <p>Resources</p>
```
Comment (michey85) [inline src/components/Header.jsx:28]: Is this a menu? If so you shouldn't use `p` tag here.

File: src/components/Main.jsx [hunk 5]
```diff
@@ -73,39 +73,41 @@ function Main() {
...
+            </div>
+            <Feature
+              title="Brand Recognition"
+              text="Boost your brand recognition with each
+            click. Generic links don't mean a thing. Branded links help instil confidence in your
+            content."
+            />
+          </section>
+          <hr className={style.line} />
+          <section className={`${style.features} ${style.secondChild}`}>
```
Comment (michey85) [inline src/components/Main.jsx:89]: Use a library for class names.

File: src/components/Main.jsx [hunk 6]
```diff
@@ -73,39 +73,41 @@ function Main() {
...
+              <Detailed />
+            </div>
+            <Feature
+              title="Detailed Records"
+              text="Gain insights into who is clicking your links. Knowing when and where people engage with
+              your content helps inform better decisions."
+            />
+          </section>
+          <hr className={style.line} />
+          <section className={`${style.features} ${style.lastChild}`}>
```
Comment (michey85) [inline src/components/Main.jsx:100]: Here as well

File: src/components/Header.module.css [hunk 7]
```diff
@@ -64,3 +68,57 @@ hr {
 .hidden {
   display: none;
 }
+
+.menu_big_screen {
+  display: none;
+  list-style-type: none;
+  padding-inline-start: 0;
```
Comment (michey85) [inline src/components/Header.module.css:75]: Why do you need any other styles here after `display: none`?

File: src/components/Header.jsx [hunk 8]
```diff
@@ -19,7 +22,26 @@ function Header() {
...
+            <li>
+              <a href="#">Pricing</a>
+            </li>
+            <li>
+              <a href="#">Resources</a>
+            </li>
+          </ul>
+        </nav>
+        <div>
+          <p>Login</p>
```
Comment (michey85) [inline src/components/Header.jsx:42]: Why logic is a `p` tag? Have you seen this as a user? I guess it has to be either button or anchor.

Comment (LazarevSlava) [reply src/components/Header.jsx]: oops

**Review by michey85 — APPROVED** [review]

**Pull request merged**